docs/usage/cloud/gitlab-installation.mdx

```diff
@@ -25,6 +25,34 @@ OpenHands requests an API-scoped token during OAuth authentication. By default,
 To restrict the agent's permissions, you can define a custom secret `GITLAB_TOKEN`, which will override the default token assigned to the agent.
 While the high-permission API token is still requested and used for other components of the application (e.g. opening merge requests), the agent will not have access to it.
 
+## Working on GitLab Issues and Merge Requests Using Openhands
+
+<Note>
+This feature works for personal projects and is available for group projects with a
+[Premium or Ultimate tier subscription](https://docs.gitlab.com/user/project/integrations/webhooks/#group-webhooks).
+
+A webhook is automatically installed within a few minutes after the owner/maintainer of the project or group logs into
+OpenHands Cloud. Deleting this webhook may require the support of All Hands AI but we are planning to improve this in
+a future release.
```

Comment:
```suggestion
OpenHands Cloud. If you decide to delete the webhook, then re-installing will require the support of All Hands AI but we are planning to improve this in a future release
```

Deleting the webhook is fine (we want users to be able to do that)

Re-installing will 100% require assistance from us, but we will have a more automatic way to do that in a future release